recipes/qt/6.x.x/test_package/conanfile.py

```diff
@@ -13,13 +13,32 @@
 
 class TestPackageConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
-    generators = "qt", "cmake", "cmake_find_package_multi", "cmake_find_package", "pkg_config", "qmake"
+    generators = "cmake", "cmake_find_package_multi", "cmake_find_package", "pkg_config", "qmake"
 
     def build_requirements(self):
         self.tool_requires("cmake/3.25.3")
         if self._meson_supported():
             self.tool_requires("meson/1.0.1")
 
+    def generate(self):
+        path = self.deps_cpp_info["qt"].rootpath.replace("\\", "/")
```

Comment:
```suggestion
        path = self.dependencies["qt"].package_folder.replace("\\", "/")
```
deps_cpp_info is Conan 1.x